src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -310,12 +304,12 @@ const GeneralInfoStep = ({ btnsBox }) => {
             />
           </Stack>
           <TextField
-            error={useFormErrors.professionalSummary}
+            error={Boolean(useFormErrors.professionalSummary)}
```

Comment:
```suggestion
            error={(useFormErrors.professionalSummary)}
```
Using Boolean() breaks length-based validation messages, because the original string gets converted to true/false, and the helperText will no longer work as expected.

<img width="450" alt="Знімок екрана 2025-05-01 о 12 07 34" src="https://github.com/user-attachments/assets/e58c1434-f4e5-4e5b-8565-9fc44b7ee748" />

The TextField component from MUI automatically treats any truthy value (like a string) as true for the error prop. So you don't need to explicitly convert your error value to a boolean.

Reply:
Ok, i will fix it now. Thank you)